weekly-reports/reports.md

```
@@ -0,0 +1,124 @@
+  <img src="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png" width="80" alt="GitHub Logo" />
+  <h1 align="center">Weekly Team Report</h1>
+  <p align="center">
+    <strong>Agile Scrum Master Project</strong>
+    <br />
+    <a href="../README.md">Return to Main README</a>
+  </p>
+</div>
+
+<hr />
+
+## <img src="https://github.com/Chetan-s-345.png?size=50" width="50" style="border-radius: 50%; vertical-align: middle;" /> Chetan (Team Lead)
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)